.eslintrc

```diff
@@ -1,2 +1,8 @@
 ---
 extends: ./node_modules/builder-docs-archetype/config/eslint/.eslintrc-config
+
+rules:
```

Comment:
I'll fix the casing & filenames, my apologies. The other rules I didn't mean to commit, was having issues with my editor & false-positives. I'll adjust these!